_docs/setup/kubernetes/quick-start.md

```diff
@@ -165,6 +165,46 @@ Output should be:
 [...] --admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota,NodeRestriction,Priority [...]
 ```
 
+### Azure
+
+You need to use `ACS-Engine` to deploy you cluster. After following [these instructions](https://github.com/Azure/acs-engine/blob/master/docs/acsengine.md#install) to get and install the `acs-engine` binary, use the following command to download Istio `api model definition`:
+
+```bash
+wget https://raw.githubusercontent.com/Azure/acs-engine/master/examples/service-mesh/istio.json
+```
+
+Use the following command to deploy your cluster using the `istio.json` template. You can find references to the parameters in the [official docs](https://github.com/Azure/acs-engine/blob/master/docs/kubernetes/deploy.md#step-3-edit-your-cluster-definition).
+
+| Parameter                           | Expected value             |
+|-------------------------------------|----------------------------|
+| `subscription_id`                     | Azure Subscription Id      |
+| `dns_prefix`                          | Cluster DNS Prefix         |
+| `location`                            | Cluster Location           |
+
+```bash
+acs-engine deploy --subscription-id <subscription_id> --dns-prefix <dns_prefix> --location <location> --auto-suffix --api-model istio.json
+```
+
+After a few minutes you should find your cluster on your Azure subscription in a resource group called `<dns_prefix>-<id>`. Let's say my `dns-prefix` is `myclustername`, a valid resource group and unique cluster id would be `mycluster-5adfba82`. Using this `<dns_prefix>-<id>` cluster id you can copy your `kubeconfig` file to your machine from the `_output` folder generated by `acs-engine`:
+
+```bash
+cp _output/<dns_prefix>-<id>/kubeconfig/kubeconfig.<location>.json ~/.kube/config
+```
+
+For example:
+```bash
+cp _output/mycluster-5adfba82/kubeconfig/kubeconfig.westus2.json ~/.kube/config
+```
+
+To check if the right Istio flags were deployed, use:
+```bash
+kubectl describe pod --namespace kube-system $(kubectl get pods --namespace kube-system | grep api | cut -d ' ' -f 1) | grep admission-control
+```
+You should see `MutatingAdmissionWebhook` and `ValidatingAdmissionWebhook` flags:
+```bash
+      --admission-control=...,MutatingAdmissionWebhook,...,ValidatingAdmissionWebhook,...
```

Comment:
It might be helpful to note that the `--admission-controls` have to be in a particular order and to link users to a working example. [here](https://github.com/Azure/acs-engine/tree/master/examples/service-mesh)